src/jesse_validator_draft3.erl

```diff
@@ -1051,3 +1051,14 @@ maybe_external_check_value(Value, State) ->
     Fun ->
       Fun(Value, State)
   end.
+
+maybe_external_check_format(Value, Format, State) ->
+    case jesse_state:get_external_format_validator(Format, State) of
+        undefined -> State;
+        Fun when is_function(Fun, 1) ->
```

Comment:
Currently there is no error handling in `jesse_state:new`, BUT there is exception handling in `jesse:validate...`, so you will get {error, Exception} in case of wrong validator function (you can get it anyway if validator throws, so checking validators signature in `jesse_state:new` wouldn't help too much)